.github/workflows/nightlies.yml

```diff
@@ -22,8 +22,8 @@ jobs:
     runs-on: macos-latest
     steps:
       - uses: actions/checkout@v1
-      - name: brew install ponyc
-        run: brew install ponyc
+      - name: install ponyc
+        run: bash macOS-install-pony-tools.bash
```

Comment:
Won't this run `ponyup update ponyc release` from `macOS-install-pony-tools.bash`? Should it instead require `ponyc nightly`?

Reply:
We build nightlies (at this time) with ponyc release.

I think we should have a discussion about if we should change that. I think yes, we should change it, but for me it is outside of the scope of this PR.

Reply:
OK